HJM.Chip8/HJM.Chip8.CPU/Changes/AddressChange.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HJM.Chip8.CPU.Changes
+{
+    /// <summary>
+    /// Represents a change at a specific address
+    /// </summary>
+    /// <typeparam name="T">Type of the change</typeparam>
+    public class AddressChange<T> : Change<T>
```

Comment:
Yeah I think I will change the name to 'ArrayChange'. I guess what I meant by address change was a change at a specific location. This works fine for Registers, Memory, and Graphics since they are all array's. I don't think another sub class is necessary.

Reply:
Good idea, no need to make the subclass if not necessary.